src/components/timer/Timer.js

```diff
@@ -11,6 +12,7 @@ const Timer = ({startTime, secondsRemaining, store}) => {
         <button onClick={() => store.dispatch(createResetTimerAction())}>
           Reset Timer
         </button>
+        <Pie totalTime={startTime} time={secondsRemaining} />
```

Comment:
Just noticed that you had totalTime as a property here. I don't think you actually read these property anywhere in your class though.

In the class it resets the totalTime onto the start during mount. If you want to use this, you would need to chase the method to 
```
const Pie = ({time, totalTime}) => {
  // Put code from render() method here, replacing `this.state.totalTime` with `time`
}

export default Pie
```

My recommendation to replace this.state.totalTime might of been wrong, you likely want to replace it with just the prop totalTime.

Reply:
Yeah, this is an area of interest. The situation: need to receive and store a "total time." In the Timer component there is startTime, which is valued in the form of CPU clock time (as seconds). Further inspection needed.